### PR TITLE
fix(node): Implement os.cpus()

### PR DESCRIPTION
**Pull request by littledivy**
Fixes https://github.com/denoland/deno/issues/21666

Zero added dependency and tries to match the libuv implementation

### CODE REVIEW

File: ext/node/ops/os/cpus.rs [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+      &mut cpu_speed as *mut _ as *mut libc::c_void,
+      &mut cpu_speed_size,
+      std::ptr::null_mut(),
+      0,
+    );
+
+    if cpu_speed == 0 {
+      // use a fixed value for frequency on arm64
+      cpu_speed = 2400000000;
+    }
```
Comment (ry) [inline ext/node/ops/os/cpus.rs:69]: can you link to some documentation here? also use underscores in big values to clarify: `2_400_000_000`

File: ext/node/ops/os/cpus.rs [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+  pub nice: u64,
+  pub sys: u64,
+  pub idle: u64,
+  pub irq: u64,
+}
+
+#[derive(Debug, Default, Serialize, Clone)]
+pub struct CpuInfo {
+  pub model: String,
+  pub speed: u64,
```
Comment (ry) [inline ext/node/ops/os/cpus.rs:17]: add a comment or rename variable to specify units

**Review by ry — COMMENTED** [review]
nice!

File: ext/node/ops/os/cpus.rs (hunk 2 above)
Comment (littledivy) [reply ext/node/ops/os/cpus.rs:17]: added doc comment above. can't rename because its part of the public Node API https://nodejs.org/api/os.html#oscpus

**Review by ry — APPROVED** [review]
LGTM

**Pull request merged**